### ISSUE TITLE
Is it possible to install the Pathways WebPart on more than one Site-Collection

### ISSUE THREAD

**Issue by Webritter** [pnp/custom-learning-office-365#413]
We have successfully  installed the WebPart in our main site-collection

We like it and other Site-Collections-Owner wants to install the webpart on their site too.

Is ist possible to install the WebPart and the Custom-List on more than one Site-Collection?

Kind regards from austria,
Helmut

**Comment by juliemturner** [comment]
Based on your question I suspect that you may have a very custom installation going.  When you say you've:
>We have successfully installed the WebPart in our main site-collection
What exactly does that mean?

Learning pathways is a tenant wide solution, by that I mean that you should have a site collection dedicated to learning pathways as the "main" learning pathways site.  The solution (spfx package) should be deployed to the tenant app catalog and as part of the solution it uses tenant app properties that allow the app to be added into multiple site collections which then surfaces the viewer web part to be added to pages in those site collections where the app has been activated.

Long story short no, you cannot have more than on "main" learning pathways site collection.

**Comment by juliemturner** [comment]
I'm going to close this issue as answered. If you have further issues, please feel free to create a new issue and reference this one.
